Zilon.Core/Zilon.Emulation.Common/AutoplayEngineBase.cs

```diff
@@ -1,63 +1,59 @@
 ﻿using System;
-using System.Linq;
 using System.Threading.Tasks;
 
 using Microsoft.Extensions.DependencyInjection;
 
-using Zilon.Bot.Players;
 using Zilon.Bot.Sdk;
 using Zilon.Core.PersonModules;
 using Zilon.Core.Persons;
-using Zilon.Core.Players;
-using Zilon.Core.Scoring;
 using Zilon.Core.Tactics;
+using Zilon.Core.Tactics.Behaviour;
+using Zilon.Core.World;
 
 namespace Zilon.Emulation.Common
 {
-    public abstract class AutoplayEngineBase<T> where T : IPluggableActorTaskSource
+    public abstract class AutoplayEngineBase<T> where T : IPluggableActorTaskSource<ISectorTaskSourceContext>
```

Comment:
Codacy found an issue: ['T' is not used in the class.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)